src/app/components/Tooltip/index.tsx

```diff
@@ -1,10 +1,15 @@
 import React, { useCallback, useState } from 'react';
 
+import { isIOS } from 'react-device-detect';
 import styled from 'styled-components';
 
 import Popover, { PopoverProps } from '../Popover';
 
-const TooltipContainer = styled.div<{ wide?: boolean }>`
+const TooltipContainer = styled.div<{ wide?: boolean; small?: boolean }>`
```

Comment:
Use mobile-first approach.

Reply:
@WeiLi512 Regarding this point, It seems we only update width for risk ratio tooltips. So I think we also need to this attribute

Reply:
Then just hard code the media query scope.

![image](https://user-images.githubusercontent.com/50083174/128674315-1d834d9e-3dba-487b-84e0-aacb9c8cbee1.png)